### PR TITLE
added html structure, CSS style and public files

### PR DESCRIPTION
**Pull request by Okputu**
Break down of PR
1. HTML STRUCTURE 
 -  added basic boiler plate of the design structure which include header, section(for body) and footer(for footer elements).
 - also added a style link to external CSS in the head tag
2. CSS style 
 - added the required style to comment the html structure according to the specifications of the design
 - also added some personal general resets.
 - normalize.css(need styles) was also used where applicable. 
3. Public files
 - design images were added and all fonts needed for the website were downloaded

4. Screenshots for Supported Viewports using responsively app.
 --Desktop
Index - Full - Laptop with HiDPI screen - 2021-04-02 at 6 35 57 PM

 --Tablet
Index - Full - Tablet - 2021-04-02 at 6 53 31 PM

 --Iphone X
Index - Full - iPhone X - 2021-04-02 at 6 35 57 PM

5. Furthermore, all reviewed changes have been update also
6. The site is hosted on netlify  - https://cocky-mirzakhani-8e971c.netlify.app
7. other requirements will be added during other PR

### CODE REVIEW

**Review by bahdcoder — DISMISSED** [review]
Everything actually looks pretty awesome. Easy to understand code, semantic html. Well done !

File: index.html [hunk 1]
```diff
@@ -1,11 +1,236 @@
...
+                My full name is Okputu, Okputu Emmanuel. I am a christian from
+                Nigeria and I hold a B.A in Economics. Currently, I work as a
+                Software developer for Active Code Business Hub. In my spare
+                time, I indulge in learning new things and Exercising. I am
+                passionate about web development and I look forward to working
+                with passionate individuals.
+              </p>
+            </div>
+          </div>
+          <hr class="first_hr" />
```
[the remark this answers is not in the document]
Comment (Okputu) [reply index.html:51]: Thanks for the feedback changes will be updated.

File: index.html [hunk 2]
```diff
@@ -1,11 +1,236 @@
...
+          <div class="education">
+            <div class="education_details">
+              <h2>Education</h2>
+              <h3>BA Economics — Modern Sciences and Arts University</h3>
+              <small>Graduated 2016</small>
+            </div>
+          </div>
+          <hr class="third_hr" />
+        </div>
+        <div class="resume_info_two">
```
[the remark this answers is not in the document]
Comment (Okputu) [reply index.html:107]: Thanks for the feedback changes will be updated.

**Review by bahdcoder — APPROVED** [review]
Everything looking smooth and perfect. Nice work sir. Thanks for sharing. One thing I noticed is that the fonts are not the right ones. Maybe you can check that? Find out what happened. Thanks !

**Pull request merged**